src/activity/components/histogram/histogram.jsx

```diff
@@ -1,35 +1,41 @@
+/* eslint no-return-assign: 0 */
 import { h } from 'preact'
 import cx from 'classnames'
 import _mean from 'lodash/mean'
-import { rgba } from 'polished'
-
-import { MONTH_NAMES } from 'src/constants'
+import { rgba, stripUnit } from 'polished'
+import { scalePow } from 'd3-scale'
+// import { MONTH_NAMES } from 'src/constants'
 
 import Bars from './bars'
 import styles from './histogram.scss'
 
 import sassVars from 'variables.scss'
 
-const shorten = m => (m).substring(0, 3)
-const avgToColor = (d, m) => rgba(sassVars.blue, _mean(m) / 100)
+// const shorten = m => (m).substring(0, 3)
+// const displayLabel = m => `${shorten(MONTH_NAMES[i])} ${year}`
+// const displayLabel = (m, y) => `${m + 1}/${y}`
+// const displayLabel = (m, y) => `${m + 1}`
+const displayLabel = (m, y) => ``
 
-const Histogram = ({ data, margin = 1, className }) => {
+const Histogram = ({ data, max, margin = 1, className }) => {
+  const totalMonths = Object.keys(data).reduce((yy, y) => yy += Number(data[y].length), 0)
+  const yScale = scalePow().domain([0, max]).range([0, 100]).exponent(0.25)
```

Comment:
Adding a labeled vertical axis will be crucial on later steps, especially since we are not using a linear scale.